net-nds/phpldapadmin/phpldapadmin-1.2.6.6.ebuild

```
@@ -0,0 +1,59 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit webapp
+
+MY_PN="phpLDAPadmin"
+DESCRIPTION="phpLDAPadmin is a web-based tool for managing all aspects of your LDAP server"
+HOMEPAGE="https://github.com/leenooks/phpLDAPadmin"
+SRC_URI="https://github.com/leenooks/${MY_PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+KEYWORDS="~alpha ~amd64 ~ia64 ~ppc ~sparc ~x86"
+IUSE=""
+
+RDEPEND="
+	>=dev-lang/php-8.0[hash(+),ldap,session,xml,nls]
+	dev-libs/openssl
```

```suggestion
	dev-libs/openssl:=
```
This way, this package will get reinstalled when switching from openssl:0/1.1 to openssl:0/3, which will help with the conditional patching.